pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
@lavalamp OK. I did not notice there is an import cycle by my eye. :( The unversion pkg is named to `client`.

>  Sorry, I can't seem to find the official go documentation of this feature, but here is a decent explanation, scroll down to "black-box test package" https://splice.com/blog/lesser-known-features-go-test/

The official doc is at https://golang.org/cmd/go/#hdr-Test_packages. 

It is the same as I mentioned 3 replies above.